.github/CODEOWNERS_baseline_errors.txt

```diff
@@ -1,30 +1,22 @@
 jeremymeng is not a public member of Azure.
 mpodwysocki is not a public member of Azure.
-xirzec is not a public member of Azure.
 minhanh-phan is not a public member of Azure.
 anilba06 is not a public member of Azure.
 dpwatrous is not a public member of Azure.
 There are no owners defined for CODEOWNERS entry.
 danielav7 is not a public member of Azure.
-bgams is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
 miguhern is not a public member of Azure.
 RoyHerrod is not a public member of Azure.
 ajpeacock0 is not a public member of Azure.
-Mrayyan is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
 shwali-msft is not a public member of Azure.
 allchiang-msft is not a public member of Azure.
 phermanov-msft is not a public member of Azure.
 ilyapaliakou-msft is not a public member of Azure.
 abkolant-MSFT is not a public member of Azure.
-pjohari-ms is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
 olivakar is not a public member of Azure.
 KarishmaGhiya is not a public member of Azure.
 schaabs is not a public member of Azure.
 EmmaZhu is not a public member of Azure.
-wangyuantao is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
```

Comment:
Not sure why those errors are coming, seems those are all valid users.

Reply:
@lirenhe : They are not valid due to organziation or permission mismatches.   wangyuantao for example, has the correct public orgs but does not have write permissions.